extra_docs/jfs/README.md

```diff
@@ -0,0 +1,209 @@
+# Jexia Fileset Module (JFS)
+
+JFS is used for the uploading, fetching and manipulating files as any other user data. Basically, fileset has the same 
+functionality as dataset, but provides a way to maintain large data objects (files).  
```

Comment:
suggestion: _Basically, ~~fileset has the same functionality as dataset~~ **fileset is a dataset**, but provides..._

Reply:
👍